packages/component/src/Composer.js

```diff
@@ -255,8 +255,8 @@ class Composer extends React.Component {
         renderMarkdown,
         scrollToEnd,
         store,
-        userID,
-        username,
+        userID: _userID, // We want to remove userID and username from propsForLogic
```

Comment:
is this a //TODO?

Reply:
No, this is required to make eslint works, so I added more explanation over there.

The idea is like this,

```js
const r1 = { a: 1, b: 2, c: 3, d: 4, e: 5 };
const { a, b, ...r2 } = r1;
```

Goal: r2 should only have { c: 3, d: 4, e: 5 }, no `a` and `b`.

Non-goal: we don't really care if `a` and `b`.